src/js/components/Notification/Notification.js

```diff
@@ -137,9 +137,11 @@ const Notification = ({
     actions = actionsProp.map((action, index) => (
       <NotificationAnchor
         key={action.label}
-        // create space between first anchor and text content
+        // create space between first anchor and text content and next anchor
         margin={
-          index === 0 && (message || title) ? { left: 'xsmall' } : undefined
+          index >= 0 && (message || title)
```

Comment:
seems like we want the margin to apply to all anchors, and the first anchor if there is a message or a title.

```
let margin;
if (index === 0 && (message || title)) margin = { left: 'xsmall' };
else if (index > 0) margin = { left: 'xsmall' };

...
 margin={margin}